CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## Whats New
+* 2018-06-08 - BTH - Fixed component support for ReactiveForms, specifically for 'disabled' and 'value'.
 * 2018-06-01 - TJM - Updated colorpicker to use ngZone and added a few missing settings. This effects the constructor so may effect those using AOT.
```

Comment:
Can you add a note like this for any of the components you refactored to ngZone. I.E. touched the constructor.
As it requires an AOT recompile so were asked to explicitly note those.

Reply:
Just merging these changes in now - my branch was taken before this.

Reply:
Not sure I understand the comment about requires an AOT recompile, do you have any information on that?  Is this Ming.le / homepages trying to drop angular components in?

Reply:
yes its mingle / home pages dropping in. my understanding is that if we change any component constructors they would like to know as they need to do something on their end @pwpatton may have a better understanding. I just think of it as making a note if we add something to a contructor / as we make the ngone changes so they can see that in the release notes.